Xamarin.Forms.Platform.UAP/SliderStyle.xaml

```diff
@@ -0,0 +1,267 @@
+                      Margin="{ThemeResource SliderHeaderThemeMargin}"
+                      Content="{TemplateBinding Header}"
+                      ContentTemplate="{TemplateBinding HeaderTemplate}"
+                      FontWeight="{ThemeResource SliderHeaderThemeFontWeight}"
+                      TextWrapping="Wrap" />
+						<Grid x:Name="SliderContainer" Background="Transparent" Grid.Row="1" Control.IsTemplateFocusTarget="True">
+							<Grid x:Name="HorizontalTemplate" MinHeight="44">
+								<Grid.ColumnDefinitions>
+									<ColumnDefinition Width="Auto" />
+									<ColumnDefinition Width="Auto" />
+									<ColumnDefinition Width="*" />
+								</Grid.ColumnDefinitions>
+								<Grid.RowDefinitions>
+									<RowDefinition Height="18" />
+									<RowDefinition Height="Auto" />
+									<RowDefinition Height="18" />
+								</Grid.RowDefinitions>
+								<Rectangle x:Name="HorizontalTrackRect"
+                      Fill="{TemplateBinding Background}"
+                      Height="{ThemeResource SliderTrackThemeHeight}"
+                      Grid.Row="1"
+                      Grid.ColumnSpan="3" />
+								<Rectangle x:Name="HorizontalDecreaseRect"
+                      Fill="{TemplateBinding Foreground}"
+                      Grid.Row="1" />
+								<TickBar x:Name="TopTickBar"
+                    Visibility="Collapsed"
+                    Fill="{ThemeResource SystemControlForegroundBaseMediumLowBrush}"
+                    Height="{ThemeResource SliderOutsideTickBarThemeHeight}"
+                    VerticalAlignment="Bottom"
+                    Margin="0,0,0,4"
+                    Grid.ColumnSpan="3" />
+								<TickBar x:Name="HorizontalInlineTickBar"
+                    Visibility="Collapsed"
+                    Fill="{ThemeResource SystemControlBackgroundAltHighBrush}"
+                    Height="{ThemeResource SliderTrackThemeHeight}"
+                    Grid.Row="1"
+                    Grid.ColumnSpan="3" />
+								<TickBar x:Name="BottomTickBar"
+                    Visibility="Collapsed"
+                    Fill="{ThemeResource SystemControlForegroundBaseMediumLowBrush}"
+                    Height="{ThemeResource SliderOutsideTickBarThemeHeight}"
+                    VerticalAlignment="Top"
+                    Margin="0,4,0,0"
+                    Grid.Row="2"
+                    Grid.ColumnSpan="3" />
+								<Thumb x:Name="HorizontalThumb"
+                  Background="{ThemeResource SystemControlForegroundAccentBrush}"
+                  Style="{StaticResource SliderThumbStyle}"
+                  DataContext="{TemplateBinding Value}"
+                  Height="24"
+                  Width="8"
+                  Grid.Row="0"
+                  Grid.RowSpan="3"
+                  Grid.Column="1"
+                  AutomationProperties.AccessibilityView="Raw" />
+								<Thumb x:Name="HorizontalImageThumb"
```

Comment:
@jassmith IIRC, the only change from the default style was adding the HorizontalImageThumb.